docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -974,6 +976,8 @@ Default templates for project boards:
 - `ENABLED_ISSUE_BY_LABEL`: **false**: Enable issue by label metrics with format `gitea_issues_by_label{label="bug"} 2`.
 - `ENABLED_ISSUE_BY_REPOSITORY`: **false**: Enable issue by repository metrics with format `gitea_issues_by_repository{repository="org/repo"} 5`.
 - `TOKEN`: **\<empty\>**: You need to specify the token, if you want to include in the authorization the metrics . The same token need to be used in prometheus parameters `bearer_token` or `bearer_token_file`.
+- `ESTIMATE_COUNTS`: **false**: Estimate counts for statistics instead counting directly.
+- `STATISTICS_TTL`: **5m**: Cache summary statistics for this period of time.
```

Comment:
It's a little strange to put these 2 new options under `metrics` which is used for `/metrics` endpoint.

I would prefer to avoid introducing new options, and just tell users that these metrics are estimated. The reason is that the accurate count number of `action`/`star`/`user`/etc do not help admin really. If you read a count number  1000 at first time, next minute it may become 1100, there is no really accurate and meaningful count number indeed. So in the end there is no difference between providing an estimated number or an accurate number.

The UI can just tell admin that these numbers are estimated, it should be enough.

Reply:
The statistics are calculated in two places.

* Admin Dashboard
* Metrics Endpoint

The options under `[metrics]` relate to the metrics endpoint.

The options under `[ui.admin]` relate to the admin dashboard endpoint.

Reply:
Sorry I misread the section. But I am still curious why the accurate number should be kept, it seems no sense.